__device-tests__/helpers/test-data.js

```diff
@@ -87,3 +87,21 @@ exports.imageCompletehtml = `<!-- wp:image {"id":1} -->
 <!-- wp:paragraph -->
 <p>The finer continuum interprets the polynomial rabbit. When can the geology runs? An astronomer runs. Should a communist consent?</p>
 <!-- /wp:paragraph -->`;
+
+exports.imageCaption = `C'est la vie my friends`;
```

Comment:
@marecar3 these attributes are already exported above this, you can remove this addition and just use the above.

Reply:
Thanks @JavonDavis, removed!